test/test_from_scratch.py

```diff
@@ -279,7 +279,7 @@ def test_content_of_summary_not_empty(s3):
         ]
     )
     # Checking if summary is populated
-    for i in range(300):
```

Comment:
would be nice to validate in an account with AWS Ent Support and a couple of cases if 3 s is enough for testing